WORKSPACE

```diff
@@ -67,11 +60,19 @@ bind(
     actual = "//tools:java_toolchain",
 )
 
-load("@rules_java//java:repositories.bzl", "rules_java_dependencies", "rules_java_toolchains")
-
-rules_java_dependencies()
-
-rules_java_toolchains()
+"""
+The rules_proto contains the javalite_toolchain needed for the java_lite_proto_library rule used in the
+model module.
+"""
+http_archive(
+    name = "rules_proto",
+    sha256 = "602e7161d9195e50246177e7c55b2f39950a9cf7366f74ed5f22fd45750cd208",
+    strip_prefix = "rules_proto-97d8af4dc474595af3900dd85cb3a29ad28cc313",
+    urls = ["https://github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz"],
+)
+load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
```

Comment:
empty line above

Reply:
I think I might have given the opposite advice above. :)

@jcqli please pick the approach that's consistent throughout this file and make sure this & the code above matches.

Reply:
Done, added empty lines